src/Board.js

```diff
@@ -1,9 +1,10 @@
 /* Represents the board state at a point in time */
 class Board 
 {
-    constructor(adjacencyMatrix)
+    constructor(adjacencyMatrix, indexRenderer)
     {
         this.cells = adjacencyMatrix;
+        this.renderIndex = indexRenderer;
```

Comment:
[Call a smeerp a smeerp.](https://tvtropes.org/pmwiki/pmwiki.php/Main/CallARabbitASmeerp) EDIT: This suggestion is a lie don't add it.
```suggestion
        this.indexRenderer = indexRenderer;
```

Reply:
I lied. Didn't realize this is actually the function. Something something RenderIndex something something I'm super inconsistent anyway and will fix it when I do styling branches in the future and make a decision about C# or JS style syntax, because clearly this project hasn't.